contracts/deployment_scripts/testnet/layer2/001_whitelist_tokens.ts

```diff
@@ -0,0 +1,95 @@
+import {HardhatRuntimeEnvironment} from 'hardhat/types';
+import {DeployFunction} from 'hardhat-deploy/types';
+import { Receipt } from 'hardhat-deploy/dist/types';
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+    const l1Network = hre.companionNetworks.layer1;
+    const l2Network = hre; 
+
+    const l1Accounts = await l1Network.getNamedAccounts();
+    const l2Accounts = await l2Network.getNamedAccounts();
+
+    const HOCDeployment = await l1Network.deployments.get("HOCERC20");
```

Comment:
This assumes the HOCERC20 was deployed in the L1, yeah ?

Reply:
yep when they are removed along with the old bridge this won't be necessary.